cmd/init/init.go

```diff
@@ -189,31 +187,31 @@ func getResource(ctx context.Context, namespace, k8sContext string) (*model.K8sO
 		namespace = client.GetContextNamespace(k8sContext)
 	}
 
-	r, err := askForResource(ctx, namespace, c)
+	app, err := askForRunningApp(ctx, namespace, c)
```

Comment:
I'm looking at this from the viewpoint of how large the interface is. In this case it looks like we need a much smaller set of functionality then is defined by the interface. I believe we simply need something that implements `PodSpec()` if embedding is used.